### DISCUSSION TITLE
Mantine UI V7

### DISCUSSION THREAD

**Discussion by Voltz45** [icflorescu/mantine-datatable#416] [Q&A]
Hi is there any news about implementing the mantine datatable with mantine V7?

**Comment by icflorescu** [comment]
Hey, @Voltz45, thanks for asking. There's an issue tracking this: https://github.com/icflorescu/mantine-datatable/issues/420

TLDR: Upgrading the project to Mantine V7 requires some significant work, I'll do it at some point, but I could really use some [sponsorship](https://github.com/sponsors/icflorescu) from companies that are using it in their products.

**Reply by ecatugy** [reply]
You deserve, I will do my part to help you, I hope you can migrate soon, a hug.

**Reply by icflorescu** [reply]
Any amount would be appreciated 🙏

**Comment by icflorescu** [comment]
### V7.1.0-alpha.5 is here 😅🎉

[![Mantine DataTable V7](https://github.com/icflorescu/mantine-datatable/assets/581999/0be62c66-978e-4ecc-9abe-0dad3254c94d)](https://icflorescu.github.io/mantine-datatable/)

Be advised that it comes with a few [breaking changes](https://github.com/icflorescu/mantine-datatable/blob/main/CHANGELOG.md#710-alpha1-2023-10-30), as expected, since Mantine came up with lots of those.

The amount of effort spent to rewrite the entire documentation website with Next.js 14 was gruesome, but I had to do it in order to make sure the component works in every example.

Contributing should also be a bit easier now, since I took the chance to simplify the repo structure; it's not a monorepo anymore, but a pretty straightforward Next.js application. See more details [here](https://icflorescu.github.io/mantine-datatable/contribute-and-support/#become-a-code-contributor).

If everything go as planned and we're not bumping into any critical bugs, I'm planning to release the final version in a day or two. Please help me by testing everything works and looks good in the documentation website and/or in your test apps.

Thank you all again for your patience and for your support!
And special thanks to the ones who willing to sponsor part of my effort - you're awesome! 🙏🙏🙏
